tools/admin/batch_invite.py

```diff
@@ -0,0 +1,36 @@
+from github import Github
+import os
+
+token = os.environ.get("TOKEN")
```

Comment:
`GH_TOKEN` would be nice, for consistency with the `gh` tool, and make it a little less likely to conflict with any other use of the more generic `TOKEN` name.

https://cli.github.com/manual/gh_help_environment

That would also be consistent with your `batch_comment.py`

Reply:
Fixed.